tests/electron/modules/calendar_spec.js

```diff
@@ -153,19 +166,6 @@ describe("Calendar module", () => {
          * RRULE TESTS:
          * Add any tests that check rrule functionality here.
          */
-	describe("sliceMultiDayEvents", () => {
```

Comment:
why is that removed?

Reply:
Because as you can see below this test is the exact same test, so it was duplicate and therefor I removed it